errors/errors.go

```diff
@@ -11,19 +15,98 @@ import (
 	"github.com/miekg/dns"
 )
 
+var log = clog.NewWithPlugin("errors")
```

Comment:
This is different from how log messages are generated by the existing `errors` plugin. 

Currently,  log messages are generated by the following line (https://github.com/coredns/coredns/blob/2fc3f5e0b10ad4e79d952d2ac2bc8d4e7534f4e1/plugin/errors/errors.go#L23)
`
		clog.Errorf("%d %s %s: %v", rcode, state.Name(), state.Type(), err)
`

Your change (see L18, L49, L53) would generate log messages in a different format. Having said that, this seems to be typically the way logs are generated.

So, is your intention to make log messages generated by the `errors` plugin to be consistent with other log messages?  I believe this is a good goal!!

Unfortunately, L18 will cause all log messages to be generated with `errors` in `plugin name` field. This is rather confusing.

Please consider either:

1) Fix it so that the `plugin name` field correctly reflects the name of the plugin that generate the error. If this  cannot be done right now, please clearly document this behavior.

2) Revert back to the format generated by the existing `errors` plugin.

Do you have some other suggestions in mind?

Reply:
1) Yes, I agree that using this opportunity to change the error log to the same format is good!!

2) The plugin name in the log is meant to reflect the origin of the error. `plugin/errors` just happens to be the plugin generating the log message.